source/developer/developer-flow.md

```diff
@@ -10,7 +10,7 @@ Here's a general workflow a Mattermost developer working on our [mattermost-serv
 1. Take a look at the [Repository structure](https://docs.mattermost.com/developer/developer-flow.html#repository-structure) to find out where to look for what you're working on.
 2. On your fork, create a branch `PLT-####` where #### is the ticket number if it is a [Jira](https://mattermost.atlassian.net) ticket, or `GH-####` if it is a GitHub Issue without a Jira ticket.
 3. Make the code changes required to complete your ticket, making sure to write or modify unit tests where appropriate.
-4. To test your changes, run `make run` from the root directory of the server respository. This will start up the server and a watcher process that will build any changes to the client as you make them. To get changes to the server it must be restarted with `make restart-server`. Your server will be running at `http://localhost:8065`.
+4. To run your code, run `make run` from the root directory of the server respository. This will start up the server and a watcher process that will build any changes to the client as you make them. To get changes to the webapp server it must be restarted with `make restart`. Your webapp server will be running at `http://localhost:8065`.
 5. Once everything works to meet the ticket requirements, stop mattermost running `make stop` in the server repository and then run `make check-style` for check your syntax and `make test` for run the test in both repositories, server and webapp.
```

Comment:
This needs modification. `make check-style` and `make test` need to be run in both repositories if you have made changes to both. Or just the specific one you modified.